### ISSUE TITLE
lizzie0.5 "leelaz process ended"

### ISSUE THREAD

**Issue by philbat** [featurecat/lizzie#278]
hello,
On a debian system, I followed the readme.file and installed lizzie.
But, with official java 
phil@debian:~/lizzie$ /usr/local/jre1.8.0_171/bin/java -jar Lizzie.jar 
Creating config file config.txt
Creating config file persist
Leelaz process ended.
Thanks for your help

**Comment by philbat** [comment]
with "print-comms": true, I get:

phil@debian:~/lizzie$ /usr/local/jre1.8.0_171/bin/java -jar Lizzie.jar 
> 1 version
> 2 lz-analyze 10
Leelaz process ended.

**Comment by featurecat** [comment]
it's fine. I am on Ubuntu. at first i couldn't get gpu mode to work with my computer but if you follow some guides online there are packages you can install that get it to work. I'll see if I can find it for you.

**Comment by featurecat** [comment]
yeah https://askubuntu.com/questions/850281/opencl-on-ubuntu-16-04-intel-sandy-bridge-cpu the top accepted answer is what worked for me with just an Intel gpu.

**Comment by philbat** [comment]
Thank you,  it seems to work with my Intel(R) Pentium(R) CPU G860 @ 3.00GHz !!

**Comment by bvandenbon** [comment]
When I ran LeeLa Zero in the AWS cloud, and made lizzie connect to it I had the exact same issue.
I started like this:

> `git clone https://github.com/gcp/leela-zero`
  `cd leela-zero/src`
  `make`

I noticed that the resulted leelaz executable did not support the `lz-analyze` command. (Actually it's easy to check this if you manually execute the `list_commands` gtp command).

It turned out that actually the `lz-analyze` isn't supported in the last "release version" of leela zero. **So, Lizzie actually requires some kind of snapshot/beta release of leela zero.**
( see: https://github.com/gcp/leela-zero/issues/1715 )

 I should have downloaded the "next" branch instead. I will the following this in a minute or 5:

> `git clone -b next https://github.com/gcp/leela-zero`
